scripts/dockerfiles/Dockerfile.build-init

```diff
@@ -15,11 +15,13 @@ ENV PATH ${PATH}:/home/.gimme/versions/gostable.linux.arm64/bin:/home/.gimme/ver
 RUN go version
 
 ENV GO111MODULE on
-RUN go env -w GOPROXY=direct
 
 # Build init process for Fluent Bit
 COPY /init/fluent_bit_init_process.go /
 COPY /go.mod /
 COPY /go.sum /
-RUN go mod tidy \
-    && go build fluent_bit_init_process.go
+RUN ( go mod tidy \
```

Comment:
nit: the go mod tidy could run on one line/RUN statement and then the build next, to make this shorter

Reply:
do you mean go mod tidy doesn't reply on GOPROXY=direct?

Reply:
will test on the pipeline whether go mod tidy effected by GOPROXY=direct, if so, will let it has its own line